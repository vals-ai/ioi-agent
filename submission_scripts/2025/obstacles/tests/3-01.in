version https://git-lfs.github.com/spec/v1
oid sha256:5b9d3473be9aa88825ffd4da19bb14fc0d5c919ac74c5ded070c2966216bfa10
size 4789723
