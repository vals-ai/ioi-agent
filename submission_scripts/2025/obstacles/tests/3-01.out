version https://git-lfs.github.com/spec/v1
oid sha256:a6b424ec4edb3eda28adb105628f5a3651382eb435849f9ce20e9a25d20b5662
size 400036
